packages/cfd/src/features/Containers/real-account-signup-verifications.tsx

```diff
@@ -130,10 +127,7 @@ const RealAccountSignupVerifications = observer(({ onFinish }: TRealAccountSignu
     };
 
     const shouldShowPOA = () => {
-        if (JURISDICTION.LABUAN === jurisdiction_selected_shortcode && is_authenticated_with_idv_photoid) {
-            return true;
-        }
-        return !['pending', 'verified'].includes(authentication_status.document_status);
+        return isPOARequiredForMT5(account_status, jurisdiction_selected_shortcode);
```

Comment:
@likhith-deriv where do we use this file actually?

Reply:
Not sure. I have asked @shontzu-deriv regd the duplicate. Its part of their refactoring initiative

Reply:
the files in the `/features` directly are not being used anywhere in prod, it was made during cfd refactoring and basically duplicate (and sometimes renamed) file , please refrain from modifying it where possible 🙏🏼

Reply:
cc: @mahdiyeh-deriv

Reply:
The flag `'authenticated_with_idv_photoid'` u r using doesn't exist anymore as part of the `COJ-210` clickup card. Hence I have updated the logic